generate-component-docs/generate.ts

```diff
@@ -9,6 +9,14 @@ const propBlacklist = ['key'];
 const tsconfigPath = path.join(__dirname, '../tsconfig.json');
 const componentsFile = path.join(__dirname, '../lib/components/index.ts');
 
+const reactNodeType =
+  'string | number | boolean | {} | ReactElement<any, string | ((props: any) => ReactElement<any, string | ... | (new (props: any) => Component<any, any, any>)> | null) | (new (props: any) => Component<any, any, any>)> | ReactNodeArray | ReactPortal';
+
+const stringAliases: Record<string, string> = {
+  [reactNodeType]: 'ReactNode',
+  boolean: 'boolean',
```

Comment:
Why is this needed?

Reply:
This was basically already in the code. Technically it seems a Boolean is a union of true and false. This just makes it show as the word boolean.